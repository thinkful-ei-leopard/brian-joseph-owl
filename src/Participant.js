import React from 'react';
import './Participant.css';

function Participant(props) {
    return ( 
        //JSX in here
        <div></div>
    );
};

export default Participant;