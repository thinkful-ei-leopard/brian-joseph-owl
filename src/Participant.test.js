import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Participant from './Participant';

describe('<Participant />', () => {
  // smoke test
    it('renders without crashing', () => {
      const div = document.createElement('div');

      ReactDOM.render(<Participant/>, div);

      ReactDOM.unmountComponentAtNode(div);
    });
  //snapshot test
    it('renders this UI as expected', () => {
      const tree = renderer.create(<Participant />).toJSON();
      expect(tree).toMatchSnapshot();
    });
});
