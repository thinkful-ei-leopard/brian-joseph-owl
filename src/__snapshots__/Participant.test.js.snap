// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Participant /> renders this UI as expected 1`] = `
<div>
  <img
    alt="the participants avatar"
  />
  <h3 />
  <p>
    
  </p>
  <button>
    Boot
  </button>
</div>
`;
